### DISCUSSION TITLE
ルビ機能が欲しい

### DISCUSSION THREAD

**Discussion by masada-tea** [HANON-games/Shiden#7] [Ideas - 要望・アイデア]
ノベルゲームを作るにあたってやはりルビ機能があると嬉しいです。

**Comment by HANON-games** [comment]
確認が遅くなってしまいましたが、やはりルビ機能は欲しいですよね！
実は初版の時点で搭載する予定で仮実装まで済ませていたのですが、フォントによっては文字のレイアウトに問題が生じることが判明したため現状は一旦オミットしております。
技術的問題が発生しなければ Ver.1.0 にて搭載を予定しておりますので、お待ちいただけますと幸いです！

**Comment by volcani** [comment]
お返事ありがとうございます、遅くなって申し訳ありません。
ver.1.0で実装予定とのこと、了解いたしました。
開発頑張ってください！
